src/ngraph/runtime/backend.cpp

```diff
@@ -37,6 +37,9 @@ std::string runtime::Backend::s_backend_shared_library_search_directory;
 // This finds the full path of the containing shared library
 static string find_my_pathname()
 {
+#ifdef NGRAPH_STATIC_LIB_ENABLE
```

Comment:
So we cannot dyn load any other backends besides CPU and INTERPRETER if the flag is enabled ?

Reply:
Agreed. This should be fixed to work correctly.

Reply:
And what is "correctly"? There's no real documentation for what these are supposed to do.